syntax = "proto3";
package proto;

message ComplexObjectL4 {
    string var_string = 1;
    ComplexObjectL41 complexObject=2;
}

message ComplexObjectL41 {
    string var_string = 1;
    ComplexObjectL42 complexObject=2;
}
message ComplexObjectL42 {
    string var_string = 1;
    ComplexObjectL43 complexObject=2;
}
message ComplexObjectL43 {
    string var_string = 1;
    ComplexObjectL44 complexObject=2;
}
message ComplexObjectL44 {
    string var_string = 1;
    ComplexObjectL45 complexObject=2;
}
message ComplexObjectL45 {
    string var_string = 1;
    ComplexObjectL46 complexObject=2;
}
message ComplexObjectL46 {
    string var_string = 1;
    ComplexObjectL47 complexObject=2;
}
message ComplexObjectL47 {
    string var_string = 1;
    ComplexObjectL48 complexObject=2;
}
message ComplexObjectL48 {
    string var_string = 1;
    ComplexObjectL49 complexObject=2;
}
message ComplexObjectL49 {
    string var_string = 1;
    ComplexObjectL410 complexObject=2;
}
message ComplexObjectL410 {
    string var_string = 1;
    ComplexObjectL411 complexObject=2;
}
message ComplexObjectL411 {
    string var_string = 1;
    ComplexObjectL412 complexObject=2;
}
message ComplexObjectL412 {
    string var_string = 1;
    ComplexObjectL413 complexObject=2;
}
message ComplexObjectL413 {
    string var_string = 1;
    ComplexObjectL414 complexObject=2;
}
message ComplexObjectL414 {
    string var_string = 1;
    ComplexObjectL415 complexObject=2;
}
message ComplexObjectL415 {
    string var_string = 1;
}

